src/main/java/pl/tomaszdziurko/jvm_bloggers/view/login/attack/BruteForceAttackMailGenerator.java

```diff
@@ -0,0 +1,49 @@
+package pl.tomaszdziurko.jvm_bloggers.view.login.attack;
+
+import com.google.common.base.Preconditions;
+import lombok.extern.slf4j.Slf4j;
+import org.antlr.stringtemplate.StringTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import pl.tomaszdziurko.jvm_bloggers.utils.NowProvider;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * @author Adam Dec
+ * @since 0.7.0
+ */
+@Component
+@Slf4j
+public class BruteForceAttackMailGenerator {
+
+    private static final String TIME = "Time";
```

Comment:
You reason about that because BruteForceAttackMailGenerator is Spring singleton and there will not be multiple instances of it on a given classloader....or ?